arduino/opencr_arduino/opencr/libraries/turtlebot3/turtlebot3_waffle/turtlebot3_core/turtlebot3_core_config.h

```diff
@@ -80,6 +80,7 @@
 #define WAIT_SECOND                      1
 #define CHECK_BUTTON_RELEASED            2
 
+#define ACCEL_FACTOR                     -0.000598  // 2.0 * -9.8 / 32768
```

Comment:
Can you link to the document here?

Also you mentioned that we need to add scaling for the other sensors like the gyro?